src/components/Footer.js

```diff
@@ -2,6 +2,6 @@ import './Footer.css';
 
 export default function Footer() {
     return (
-        <div className="Footer">© Louie Taylor Music 2021</div>
+        <div className="Footer">© Louie Taylor Music 2023</div>
```

Comment:
This should be 2021-2023
Link to explain why here: 
https://sonet.digital/blog/general/keep-copyright-up-to-date/#:~:text=Should%20the%20Copyright%20Date%20Start%20with%20the%20Initial%20Date%20of%20Publication%3F&text=It's%20essential%20to%20update%20the,the%20site%20is%20actively%20maintained.